xbmc/addons/gui/skin/SkinTimers.dox

```diff
@@ -0,0 +1,145 @@
+/*!
```

Comment:
@jjd-uk mind taking a look at the wording/English in this file please? Pretty sure I have typos here

Reply:
Done, everything was perfectly understandable so just some minors.

Reply:
Muchas gracias,
Updated all your comments :+1: